src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => setTimeout(reject, 3000));
+
+function solution() {
+  const message = document.createElement('div');
+
+  message.classList.add('message');
+  message.textContent = 'Promise was resolved!';
+  document.body.appendChild(message);
+}
+
+function errorFound() {
+  const errorMessage = document.createElement('div');
+
+  errorMessage.classList.add('error-message');
+  errorMessage.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMessage);
+}
+
+promise1.then(solution);
+
+promise1.catch(errorFound);
```

Comment:
You can also chain then and catch
```suggestion
promise1.then(solution).catch(errorFound);
```